src/benchmark.cr

```diff
@@ -1,3 +1,47 @@
+# A Performance Benchmarking Library
+# Overview
+#
+# The Benchmark module provides methods for benchmarking Ruby code, giving
+# detailed reports on the time taken for each task.
+#
+
+# The Benchmark module provides methods to measure and report the time
+# used to execute Ruby code.
+#
+# * Measure the time to construct the string given by the expression
+#   <code>"a"*1_000_000_000</code>:
+#
+#       require 'benchmark'
+#
+#       puts Benchmark.measure { "a"*1_000_000_000 }
+#
+#   On my machine (OSX 10.10.3 on i7 2.2 Ghz) this generates:
+#
+#        0.190000   0.220000   0.410000 (  0.420185)
+#
+#   This report shows the user CPU time, system CPU time, the sum of
+#   the user and system CPU times, and the elapsed real time. The unit
+#   of time is seconds.
+
+# * Do some experiments sequentially using the #bm method:
+#
+#       require 'benchmark'
+#
+#       n = 5000000
+#       Benchmark.bm do |x|
+#         x.report { n.times do   ; a = "1"; end }
+#         x.report { 1.upto(n) do ; a = "1"; end }
```

Comment:
Thanks!

But there's still @rhysd comment, `require 'benchmark'` (note the single quotes) is not valid Crystal code. Are you sure you tested this code? I mean, not test that the documentation generates well, but that the code samples compile and work.